src/MongoDB.Driver.Core/Core/Misc/Feature.cs

```diff
@@ -276,6 +277,11 @@ public class Feature
         /// </summary>
         public static Feature MmapV1StorageEngine => __mmapV1StorageEngine;
 
+        /// <summary>
+        /// Gets the parallel collection scan feature.
+        /// </summary>
+        public static Feature ParallelCollectionScan => __parallelCollectionScan;
+
```

Comment:
I added this here for consistency but ParallelCollectionScan is never used in the test suite (the FeatureId ParallelScanCommand is used instead), so I'm not sure what to do with this.

Reply:
You're 100% correct: I hadn't noticed the FeatureId initially, After discussing this offline with Robert, I don't think we need to change this class anymore, so feel free to revert the changes to this class and resolve this conversation afterwards: sorry for the confusion!

Reply:
And [this line](https://github.com/ksadoff/mongo-csharp-driver/blob/501208fec89e0b38466b40552ea774c4241817cc/tests/MongoDB.Driver.Legacy.Tests/MongoCollectionTests.cs#L2941) in the test will ensure that the test only runs on a server with the proper version, although it's worth noting that we only use this pattern for the Legacy tests: for newer tests, we use the `RequireServer.Check().Supports(...)` pattern